src/include/lean/lean.h

```diff
@@ -451,6 +451,19 @@ static inline void lean_dec(lean_object * o) { if (!lean_is_scalar(o)) lean_dec_
 /* Just free memory */
 LEAN_EXPORT void lean_dealloc(lean_object * o);
 
+/*
+An object `o` satisfies `lean_is_ctor` if it is a member of an inductive type, barring the exceptions
+listed at https://lean-lang.org/lean4/doc/dev/ffi.html?highlight=ffi#translating-types-from-lean-to-c.
+A notable example of an exception are inductive types where all constructors have no parameters (enumeration types). 
```

Comment:
The singular/plural isn't really working here. What about:
```suggestion
One notable exception is that inductive types where all constructors have no parameters (also called _enumeration types_) is represented more efficiently, so their values do not satisfy `lean_is_ctor`. 
```